data/remnant/remnant 1 introduction.txt

```diff
@@ -721,6 +721,69 @@ mission "Remnant: Void Sprites 3"
 		system "Nenia"
 		personality staying heroic nemesis frugal uninterested
 		ship "Archon (Cloaked)" "Lifted Lorax"
+		to spawn
+			or
+				has "flagship model: Albatross"
+				has "flagship model: Gull"
+				has "flagship model: Heron"
+				has "flagship model: Ibis"
+				has "flagship model: Merganser"
+				has "flagship model: Pelican"
+				has "flagship model: Penguin"
+				has "flagship model: Peregrine"
+				has "flagship model: Petrel"
+				has "flagship model: Puffin"
+				has "flagship model: Smew"
+				has "flagship model: Starling"
+				has "flagship model: Tern"
```

Comment:
It's times like these that I think we should just add tags to ships and outfits that can be referenced as conditions.